pkg/storaged/stratis-panel.jsx

```diff
@@ -159,16 +174,28 @@ export function create_stratis_pool(client) {
                 return prepare_available_spaces(client, vals.disks).then(function (paths) {
                     const devs = paths.map(p => decode_filename(client.blocks[p].PreferredDevice));
 
-                    function create(key_desc) {
-                        return client.stratis_create_pool(vals.name, devs, key_desc).then(std_reply);
+                    function create(key_desc, adv) {
+                        let clevis_info = null;
+                        if (adv)
+                            clevis_info = ["tang", JSON.stringify({ url: vals.tang_url, adv })];
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230601-113309-0e7aeb29-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)